packages/react/test/specs/commonTests/isConformant.tsx

```diff
@@ -319,18 +329,24 @@ export default (Component, options: Conformant = {}) => {
     })
 
     test('spreads "attributes" on root', () => {
-      const role = faker.lorem.word() as AriaRole
-      const noopBehavior: Accessibility = () => ({
-        attributes: {
-          root: { role },
-        },
-      })
-
       const wrapper = mount(<Component {...requiredProps} accessibility={noopBehavior} />)
       const element = getComponent(wrapper)
 
+      expect(element.prop(IS_FOCUSABLE_ATTRIBUTE)).toBe(true)
       expect(element.prop('role')).toBe(role)
     })
+
+    test('attributes passed by consumer has more weight', () => {
```

Comment:
nit-nit: maybe we could just state supported scenario, like 'consumer's attributes override the ones provided by Stardust'